locale/en/docs/meta/topics/dependencies.md

```diff
@@ -26,7 +26,7 @@ There are several dependencies that Node.js relies on to work the way it does.
 The V8 library provides Node.js with a JavaScript engine, which Node.js
 controls via the V8 C++ API. V8 is maintained by Google, for use in Chrome.
 
-* [Documentation](https://v8docs.nodesource.com/)
+* [Documentation](https://v8.dev/)
```

Comment:
Would https://v8.dev/docs be better since it's labeled "Documentation"?

Reply:
I think it will be better, I'll change it again.